### PR TITLE
Masterbar: Fix out border & icon color on support-session

### PR DESCRIPTION
**Pull request by lupus2k**
Slack: p1723039124414569-slack-C06DN6QQVAQ

## Proposed Changes

Fix masterbar out border color & icon color for support session.

| Before | After |
| --- | --- |
| image | image |
| image | image|

## Testing Instructions

* Start a support session
* Go to `/sites` or `/read`
* Check the selected item border & icon color

### CODE REVIEW

**Review by allilevine — COMMENTED** [review]
Definitely an improvement! Should we also change the hover text and background colors?
Screen Shot 2024-08-07 at 3 21 07 PM
Screen Shot 2024-08-07 at 3 22 30 PM

I tested by opening a support session and then pasting in the Calypso branch URL.

**Comment by lupus2k** [comment]
> Definitely an improvement! Should we also change the hover text and background colors? 

Sure, I'll do it!

**Comment by lupus2k** [comment]
I have two suggestions, WDYT, @allilevine ?

Before:

https://github.com/user-attachments/assets/36e4db88-d47a-473d-a22b-c58a84a1f284

Suggestion 1:

No background change + white icon & text.

https://github.com/user-attachments/assets/6c0f1d53-4e06-4c26-8b33-345cdf0926db

Suggestion 2:

`orange-60` color background + white icon & text.
Cons: The border of the selected item will remain "orange-50" and will not look very nice (same thing happens without support session, but it is not noticeable)

https://github.com/user-attachments/assets/bb85411f-c376-40a1-a9b7-5ea7cede66e9

**Comment by lupus2k** [comment]
This is what the suggestion 2 issue show:

image

**Comment by allilevine** [comment]
> I have two suggestions, WDYT, @allilevine ?

@lupus2k I vote for Suggestion 1 because it's closer to how the non-support-session bar looks. 

It's too bad about that border issue, the `orange-60` is nice. ✨

File: client/layout/masterbar/style.scss [hunk 1]
```diff
@@ -565,6 +570,12 @@ body.is-mobile-app-view {
 
 	.is-support-session &.is-active {
 		background: var(--studio-orange-70);
+		--color-masterbar-icon: var(--studio-gray-5) !important;
+		--color-masterbar-text: var(--studio-gray-5) !important;
+	}
```
Comment (allilevine) [inline client/layout/masterbar/style.scss:576]: I'm still seeing the blue on hover on an active tab, this should remove it:

```suggestion
       --color-masterbar-highlight: var(--studio-gray-5) !important;
	}
```

File: client/layout/masterbar/style.scss [hunk 2]
```diff
@@ -565,6 +570,12 @@ body.is-mobile-app-view {
...
+
+	.is-support-session &:hover {
+		background: var(--studio-orange-50);
 	}
```
Comment (allilevine) [inline client/layout/masterbar/style.scss:581]: The tab background doesn't change on hover on an active tab on the regular bar. Here's my attempt at doing that for the support session bar:

```suggestion
.is-support-session &.is-active:hover {
		background: var(--studio-orange-70);
		color: var(--studio-gray-5);
	}
```

https://github.com/user-attachments/assets/1876e5a2-2c9a-4da3-a1c5-e267bf056001

Comment (lupus2k) [reply client/layout/masterbar/style.scss:581]: Updated [here](https://github.com/Automattic/wp-calypso/tree/fc3a63ca7a91a2605c69ec7e71e6f507edd2af5f), thanks for the suggestion, it works great :tada:

File: client/layout/masterbar/style.scss (hunk 1 above)
Comment (lupus2k) [reply client/layout/masterbar/style.scss:576]: Fixed [here](https://github.com/Automattic/wp-calypso/tree/fc3a63ca7a91a2605c69ec7e71e6f507edd2af5f)

**Review by allilevine — APPROVED** [review]
Thanks for making the hover changes!

**Pull request merged**